src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,56 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+
+    private static final DateTimeFormatter DATES = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
+    public LocalDate parseDate(String date) {
+        try {
+            return LocalDate.parse(date, DATES);
+        } catch (Exception e) {
+            throw new InvalidDataException(
+                    "Invalid date format. Only dd.MM.yyyy format is accepted"
+            );
```

Comment:
The catch block is catching a generic `Exception`, which is not a best practice. It's better to catch specific exceptions like `DateTimeParseException` to handle only the expected errors.